packages/autocertifier/src/Database.ts

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable @typescript-eslint/parameter-properties, quotes */
+
+import sqlite3 from 'sqlite3'
+import { open, Statement, Database as SqliteDatabase } from 'sqlite'
+import { Logger } from '@streamr/utils'
+import { DatabaseError, InvalidSubdomainOrToken } from '@streamr/autocertifier-client'
+import { filePathToNodeFormat } from '@streamr/utils'
+
+const logger = new Logger(module)
+
+export class Database {
+
+    private db?: SqliteDatabase
+    private createSubdomainStatement?: Statement
+    private getSubdomainStatement?: Statement
+    private getSubdomainWithTokenStatement?: Statement
+    private updateSubdomainIpAndPortStatement?: Statement
+    private getSubdomainAcmeChallengeStatement?: Statement
+    private updateSubdomainAcmeChallengeStatement?: Statement
```

Comment:
Could simplify state by creating these statements on-demand (in the functions which use these)